scripts/run_action.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -eu
```

Comment:
I've made sure `-eu` is on every action.

Reply:
Why are you changing this?

I think the explicit set below is bit safer, since you can call the script in different ways. E.g. if you do `bash run_action`, it may ignore that header.

Reply:
Ah, makes sense. I'll make them explicit then.